Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/launcher/LauncherActivityViewModel.kt

```diff
@@ -15,12 +16,14 @@ class LauncherActivityViewModel @AssistedInject constructor(
     private val updateChecker: UpdateChecker,
     dispatcherProvider: DispatcherProvider,
     private val cwaSettings: CWASettings,
+    private val encryptedPreferencesMigration: EncryptedPreferencesMigration
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
 
     val events = SingleLiveEvent<LauncherEvent>()
 
     init {
         launch {
+            encryptedPreferencesMigration.migrate()
```

Comment:
Can we migrate earlier? Can we migrate async?

Note that `launch` will be canceled when the viewmodel is destroyed. That seems risky.